package.json

```
@@ -9,15 +9,15 @@
   "scripts": {
     "dev": "ts-node-dev --no-notify -P tsconfig.server.json ./server/app.ts",
     "typeorm": "ts-node -P tsconfig.server.json ./node_modules/.bin/typeorm",
-    "db:generate": "yarn typeorm migration:generate -n",
-    "db:drop": "yarn typeorm schema:drop",
+    "db:generate": "npm run typeorm migration:generate -- -n",
+    "db:drop": "npm run typeorm schema:drop",
     "db:seed": "ts-node -P tsconfig.server.json ./db/generator/index.ts",
-    "db:migrate": "yarn typeorm migration:run",
-    "db:reset": "yarn db:drop && yarn db:migrate && yarn db:seed",
-    "both": "concurrently \"yarn dev\" \"yarn --cwd='./client' dev\"",
-    "build:client": "yarn --cwd='./client' build",
+    "db:migrate": "npm run typeorm migration:run",
+    "db:reset": "npm run db:drop && npm run db:migrate && npm run db:seed",
+    "both": "concurrently 'npm run dev' 'cd client && npm run dev'",
```

```suggestion
    "both": "concurrently \"npm run dev\" \"cd client && npm run dev\"",
```
Adding review suggestion from @paizle